src/pages/Admin/Auth/ForgotPassword/AdminForgotPassword.jsx

```diff
@@ -1,14 +1,91 @@
-import React from 'react'
-import FormNavbar from '../../../../components/Auth/Forms/Widgets/FormNavbar'
-import ForgotPasswordForm from '../../../../components/Auth/Forms/AuthForm/PasswordReset/ForgotPassword/ForgotPasswordForm'
+import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Input, Form, notification } from 'antd'
 import Layout from '../../../../components/Auth/Forms/AuthForm/PasswordReset/Layout/Layout'
 import classes from './AdminForgotPassword.module.scss'
+import { useAdminforgotPasswordMutation } from '../../../../features/slices/auth/admin/adminAuthApiSlice'
+import icon from '../../../../assets/Auth/Group 18.svg'
+import FormButton from '../../../../components/Auth/Forms/Widgets/FormButton'
 const ForgotPasswordPage = () => {
+  const [errMsg, setErrMsg] = useState('')
+  const [successMsg, setSuccessMsg] = useState('')
+  const [adminforgotPassword, { data, isLoading }] =
+    useAdminforgotPasswordMutation()
+  const openNotification = (email) => {
+    notification.success({
+      message: successMsg,
+      description: `OPT has been sent to ${email}`,
+    })
+  }
+  const navigate = useNavigate()
+  const onFinish = async (values) => {
+    const email = {
+      email: values.email,
+    }
+    try {
+      await adminforgotPassword(email).unwrap()
```

Comment:
you could pass the `email` object into the `adminforgotPassword` function like this `adminforgotPassword({ email: values.email })` instead of first declaring a variable to store it and then pass to the function.

the new variable isn't needed unless you have some manipulation to do to it, passing it directly helps us avoid unnecessary memory

Reply:
Okay